### PR TITLE
DOCSUP-9597: Add thread_name in system.stack_trace and add support DateTime64 for ver parameter

### PR DESCRIPTION
**Pull request by sevirov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation for #24124 and #23992

### CODE REVIEW

File: docs/en/engines/table-engines/mergetree-family/replacingmergetree.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ For a description of request parameters, see [statement description](../../../sq
 
 **ReplacingMergeTree Parameters**
 
--   `ver` — column with version. Type `UInt*`, `Date` or `DateTime`. Optional parameter.
+-   `ver` — column with version. Type `UInt*`, `Date`, `DateTime` or `DateTime64`. Optional parameter.
```
Comment (adevyatova) [inline docs/en/engines/table-engines/mergetree-family/replacingmergetree.md:37]:
```suggestion
-   `ver` — column with the version number. Type `UInt*`, `Date`, `DateTime` or `DateTime64`. Optional parameter.
```

File: docs/ru/engines/table-engines/mergetree-family/replacingmergetree.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ CREATE TABLE [IF NOT EXISTS] [db.]table_name [ON CLUSTER cluster]
 
 **Параметры ReplacingMergeTree**
 
--   `ver` — столбец с версией, тип `UInt*`, `Date` или `DateTime`. Необязательный параметр.
+-   `ver` — столбец с версией. Тип `UInt*`, `Date`, `DateTime` или `DateTime64`. Необязательный параметр.
```
Comment (adevyatova) [inline docs/ru/engines/table-engines/mergetree-family/replacingmergetree.md:36]:
```suggestion
-   `ver` — столбец с номером версии. Тип `UInt*`, `Date`, `DateTime` или `DateTime64`. Необязательный параметр.
```

File: docs/ru/engines/table-engines/mergetree-family/replacingmergetree.md [hunk 3]
```diff
@@ -65,4 +65,3 @@ CREATE TABLE [IF NOT EXISTS] [db.]table_name [ON CLUSTER cluster]
 -   `ver` — столбец с версией. Необязательный параметр. Описание смотрите выше по тексту.
```
Comment (adevyatova) [inline docs/ru/engines/table-engines/mergetree-family/replacingmergetree.md:65]:
```suggestion
-   `ver` — столбец с номером версии. Необязательный параметр. Описание смотрите выше по тексту.
```

File: docs/en/operations/system-tables/stack_trace.md [hunk 4]
```diff
@@ -21,61 +22,49 @@ SET allow_introspection_functions = 1;
...
+
+thread_id:   5928
+query_id:
+res:         /lib/x86_64-linux-gnu/libpthread-2.27.so
+/usr/lib/debug/.build-id/8f/93104326882d1c62c221c34e6bad1b0fefc1e5.debug
+/usr/lib/debug/.build-id/8f/93104326882d1c62c221c34e6bad1b0fefc1e5.debug
+/usr/lib/debug/.build-id/8f/93104326882d1c62c221c34e6bad1b0fefc1e5.debug
+/usr/lib/debug/.build-id/8f/93104326882d1c62c221c34e6bad1b0fefc1e5.debug
+/usr/lib/debug/.build-id/8f/93104326882d1c62c221c34e6bad1b0fefc1e5.debug
+/usr/lib/debug/.build-id/8f/93104326882d1c62c221c34e6bad1b0fefc1e5.debug
```
Comment (tavplubix) [inline docs/en/operations/system-tables/stack_trace.md:65]: Bad example

**Review by tavplubix — COMMENTED** [review]
Let's just add `thread_name: clickhouse-serv` to output examples without changing other columns

**Review by tavplubix — APPROVED** [review]

**Pull request merged**